shell/mixins/resource-fetch.js

```diff
@@ -135,17 +144,9 @@ export default {
       let incremental = 0;
 
       // get resource counts
-      if ( this.$store.getters[`${ inStore }/haveAll`](COUNT) ) {
```

Comment:
not sure if we should eliminate this check, although from the top of my head I can't find a case where this is needed... I must have added it for reason but I don't know right now...

Reply:
Any page that depends on a cluster blocks on a `clusterReady` flag, that flag is set after some initial resources have loaded and one of which is `COUNT`s. So we wouldn't be able to get to this point without it being available.

(see `loadCluster` action in `store/index`, around line 827 and 842)

Reply:
nice! Thanks for the explanation